msg/sensor_gyro_control.msg

```diff
@@ -0,0 +1,7 @@
+uint32 device_id	# unique device ID for the sensor that does not change between power cycles
```

Comment:
As this topic is meant to disappear after v1.10 is issued, what about adding a short comment saying basically "do not rely on this for custom code".

Reply:
Couldn't hurt.